rasa/nlu/test.py

```diff
@@ -422,11 +422,89 @@ def substitute_labels(labels: List[Text], old: Text, new: Text) -> List[Text]:
     return [new if label == old else label for label in labels]
 
 
+def collect_entity_errors(
+    entity_results: List[EntityEvaluationResult],
+    merged_targets: List[Text],
+    merged_predictions: List[Text],
+    error_filename: Text,
+):
+    errors = []
+
+    offset = 0
+    for entity_result in entity_results:
+        error = False
+        for i in range(offset, offset + len(entity_result.tokens)):
+            if merged_targets[i] != merged_predictions[i]:
+                error = True
+                break
+
+        if error:
+            errors.append(
+                {
+                    "text": entity_result.message,
+                    "entities": entity_result.entity_targets,
+                    "predicted_entities": entity_result.entity_predictions,
+                }
+            )
+        offset += len(entity_result.tokens)
+
+    if errors:
+        utils.write_json_to_file(error_filename, errors)
+        logger.info("Incorrect enntity predictions saved to {}.".format(error_filename))
+        logger.debug(
+            "\n\nThese intent examples could not be classified "
+            "correctly: \n{}".format(errors)
+        )
+    else:
+        logger.info("No incorrect entity prediction found.")
```

Comment:
```suggestion
        logger.info("No incorrect entity predictions found.")
```
or maybe just "Your model predicted all entities successfully" instead of double negative